src/components/DropdownMenu.js

```diff
@@ -89,8 +87,12 @@ const DropdownMenu = forwardRef(
                     }));
                     setDefaultValue(values);
                 })
-                .catch((error) => addErrorMessage(error.message))
-                .finally(() => setBusy(false));
+                .catch((error) => {
+                    console.log(error); // DEBUG
```

Comment:
`consoel.log()` is mainly used using development purpose for debugging we shouldn't be using this in production code.

Reply:
Sure, I will apply the change there.